templates/python-empty/template.config.toml

```diff
@@ -4,8 +4,12 @@ post_install_print = """
 📂 Go to your project directory:
     $ cd {project_dir}
 
+🥄 Create a virtual environment (optional, recommended):
+    $ python3 -m venv .venv
+    $ source .venv/bin/activate
+
 📦 Install Dependencies:
-    $ pip install .
+    $ pip install -r ./requirements.txt
```

Comment:
this is inconsistent with what I've seen before where we do dependency management in `setup.py`